lib/topological_inventory/api/cfme_manifest.rb

```diff
@@ -0,0 +1,28 @@
+module TopologicalInventory
+  module Api
+    class CfmeManifest
+      def self.find(version)
+        version.gsub!(".", "_")
+
+        # Find the manifest file that best matches the version supplied.
+        # Look for a full match first then iteratively check for broader
+        # and broader versions.
+        #
+        # E.g. 5.11.0.0 will check for 5.11.0.0, then 5.11.0, 5.11, then 5
+        until version.blank? || (manifest = manifest_by_version[version]).present?
+          version = version.split("_")[0...-1].join("_")
+        end
+
+        manifest
+      end
+
+      private_class_method def self.manifest_by_version
+        # List all manifests and cache the result to prevent a readdir on every API call
+        @manifest_by_version ||=
```

Comment:
Just noticed this and the only downside is we can't add new versions without bouncing the app...even so, that's probably ok since we do a redeploy anyway.

Reply:
Yeah I considered that but since they come from the filesystem on the container image I figured it would have to be rebuilt and bounced anyway.

If we want to pull them from a location which can be updated while running then we can change this.